src/backend/libs/efiling-commons/src/main/java/ca/bc/gov/open/jag/efilingcommons/model/CreateAccountRequest.java

```diff
@@ -0,0 +1,92 @@
+package ca.bc.gov.open.jag.efilingcommons.model;
+
+/**
+ * Represents a request to create an account
+ */
+public class CreateAccountRequest {
+
+    private String firstName;
```

Comment:
dumb question:

can you do this? `private String firstName, middleName, lastName;`

drawbacks?

Reply:
non blocking